src/main/java/transforms/GeocoderTransform.java

```diff
@@ -68,7 +68,19 @@ public String getAddress( String input, String addressObject ) {
         if ( outercodes.contains( addressObject ) ) {
             return (String) address.get( addressObject );
         } else if ( addressObject.equals("geographypoint") ) {
-            return address.get( "lat" ) + "," + address.get("lon");
+            String latitude = address.get( "lat" ).toString();
+            String longitude = address.get("lon").toString();
```

Comment:
Also, where do these "lat" and "lon" strings come from? Are these keys that are provided during the integration, or are they coming from user data?

Reply:
I believe these are special to the geocoder api.

Reply:
ohhh yeah you're right -- yeah probably good to constantify at the top then (ideally with a comment explaining where they come from). Same for "geographypoint"